test/core/utils/preload-cssom.js

```diff
@@ -39,7 +39,7 @@ describe('axe.utils.preloadCssom', function() {
 		var actual = axe.utils.preloadCssom({ treeRoot: treeRoot });
 		actual
 			.then(function(cssom) {
-				assert.isAtLeast(cssom.length, 2);
+				assert.isAtLeast(cssom.length, 1);
```

Comment:
We no longer have additional CSS files in test file so there is only the 1 file stylesheet now